tests/scripts/test_archive.py

```
@@ -0,0 +1,120 @@
+        print_step(8, "Cleaning up Work Item")
+        client.work_items.delete(workspace_slug, project.id, work_item.id)
+        print_success("Work Item deleted.")
+
+        # Delete project
+        print_step(9, "Cleaning up Project")
+        client.projects.delete(workspace_slug, project.id)
+        print_success("Project deleted.")
+
+        print("\n🎉 All Archive API methods tested successfully!")
+
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)